.xrefcheck.yaml

```diff
@@ -4,15 +4,9 @@
 
 # Parameters of repository traversal.
 traversal:
```

Comment:
We can delete all config fields that use their default values, since they're optional now.

The only ones we need to keep are `ignoreRefs` and `scanners.markdown.flavor`